templates/DevTools.ss

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset=utf-8/>
+    <meta name="viewport" content="user-scalable=no, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, minimal-ui">
+    <title>GraphQL IDE | Silverstripe CMS</title>
+    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/graphql-playground-react/build/static/css/index.css" />
+    <link rel="shortcut icon" href="//cdn.jsdelivr.net/npm/graphql-playground-react/build/favicon.png" />
+    <script src="//cdn.jsdelivr.net/npm/graphql-playground-react/build/static/js/middleware.js"></script>
```

Comment:
I don't think that's a safe way to execute JS. We could pin this to a specific tag and add resource integrity to the `<script>` tag to avoid the origin being tampered with. Or we can include this via our own `package.json` build. But loading random JS from untrusted sources which then have access to all data exposed under the user session isn't OK.

Reply:
Is there any reason why we can't just download the .js, .css and .png files and just put them into our repo?  That would be the simplest solution.

Reply:
Yeah I'd be OK with that as well. Inlining deps makes it more brittle to upgrade, but only slightly more so than updating a `<script>` tag. I don't feel strongly either way, as long as we don't load unvalidated stuff from an untrusted source.